### ISSUE TITLE
Location Expired

### ISSUE THREAD

**Issue by 010110a** [bilde2910/Hauk#62]
![Screenshot_2019-12-03 phpinfo()](https://user-images.githubusercontent.com/58488247/70086660-1b834980-1613-11ea-9420-723b15e43769.png)
Hello Marius,
first of all thank you for your great project. I have a small issue (well not so small actually). Everything seems to work perfectly but when I open the share link no map appears. The message on the page is "Location expired. The shared location you tried to access was not found on the server. If this link worked before, the share might have expired." On the app everything seems to work perfectly. Maybe some directory permission problem?
Thank you in advance for your help.
Cheers
Max

ps: memcached info attached

**Comment by bilde2910** [comment]
Hi! Thanks for the feedback :-)

This typically happens when the `memcached` daemon itself isn't running - the PHP extension and `memcached` itself are two separate things that interact with each other, and both must be installed and running. Can you check if it is actually running? You can check using any of these:

```bash
user@host ~$ ps aux | grep memcached
memcach+     772  0.0  0.0 409200  5752 ?        Ssl  14:12   0:03 /usr/bin/memcached -m 64 -c 1024 -l 127.0.0.1,::1 -o modern,drop_privileges

user@host ~$ sudo netstat -tulpnx | grep memcached
tcp        0      0 127.0.0.1:11211         0.0.0.0:*               LISTEN      772/memcached
tcp6       0      0 ::1:11211               :::*                    LISTEN      772/memcached
```

**Comment by 010110a** [comment]
Thank you for your super-fast reply!
Here the result:
tcp        0      0 127.0.0.1:11211         0.0.0.0:*               LISTEN      22498/memcached

**Comment by bilde2910** [comment]
Thanks for the quick response :-)

Can you try editing your Hauk config file and setting `memcached_host` to `127.0.0.1` instead of `localhost` and see if that helps?

**Comment by 010110a** [comment]
Nope :-( I got the same message. Maybe I should change something on ufw firewall? Or apache2 server?

**Comment by bilde2910** [comment]
Forwarding ports should not be necessary. I assume you're running apache2 and memcached on the same server?

You could try stopping `memcached`, then starting it manually with `memcached -m 64 -c 1024 -l 127.0.0.1,::1 -vv`. That should give you detailed connection logs as output and show what's going on behind the scenes. I don't know exactly how helpful it will be, but if something happens, it'll at least show up as a connection.

**Comment by 010110a** [comment]
Thank you again for your reply,
this is what I get:

slab class   1: chunk size        96 perslab   10922
slab class   2: chunk size       120 perslab    8738
slab class   3: chunk size       152 perslab    6898
slab class   4: chunk size       192 perslab    5461
slab class   5: chunk size       240 perslab    4369
slab class   6: chunk size       304 perslab    3449
slab class   7: chunk size       384 perslab    2730
slab class   8: chunk size       480 perslab    2184
slab class   9: chunk size       600 perslab    1747
slab class  10: chunk size       752 perslab    1394
slab class  11: chunk size       944 perslab    1110
slab class  12: chunk size      1184 perslab     885
slab class  13: chunk size      1480 perslab     708
slab class  14: chunk size      1856 perslab     564
slab class  15: chunk size      2320 perslab     451
slab class  16: chunk size      2904 perslab     361
slab class  17: chunk size      3632 perslab     288
slab class  18: chunk size      4544 perslab     230
slab class  19: chunk size      5680 perslab     184
slab class  20: chunk size      7104 perslab     147
slab class  21: chunk size      8880 perslab     118
slab class  22: chunk size     11104 perslab      94
slab class  23: chunk size     13880 perslab      75
slab class  24: chunk size     17352 perslab      60
slab class  25: chunk size     21696 perslab      48
slab class  26: chunk size     27120 perslab      38
slab class  27: chunk size     33904 perslab      30
slab class  28: chunk size     42384 perslab      24
slab class  29: chunk size     52984 perslab      19
slab class  30: chunk size     66232 perslab      15
slab class  31: chunk size     82792 perslab      12
slab class  32: chunk size    103496 perslab      10
slab class  33: chunk size    129376 perslab       8
slab class  34: chunk size    161720 perslab       6
slab class  35: chunk size    202152 perslab       5
slab class  36: chunk size    252696 perslab       4
slab class  37: chunk size    315872 perslab       3
slab class  38: chunk size    394840 perslab       2
slab class  39: chunk size    524288 perslab       2
<26 server listening (auto-negotiate)
failed to listen on TCP port 11211: Address already in use

**Comment by bilde2910** [comment]
>failed to listen on TCP port 11211: Address already in use

Did you stop the other memcached service (sudo systemctl stop memcached)?

**Comment by 010110a** [comment]
Yes I did and then I tried to restart the app a couple of times and now it works!!!!
Actually I was starting to feel guilty, hope I didn't ruin your night ;-)

All the best

**Comment by bilde2910** [comment]
Ah, don't worry, at least it works now! I'm just glad to be here to try and help :-)

Hope you'll enjoy the app

**Comment by licaon-kter** [comment]
FYI, one gets the same error message if the external .js scripts from that  CDN are blocked (eg. Via uMatrix)

**Comment by bilde2910** [comment]
>FYI, one gets the same error message if the external .js scripts from that CDN are blocked (eg. Via uMatrix)

Interesting, didn't realize that. Should be fixed now with the resolution to #67.
